### ISSUE TITLE
Unable to get past upgrade screen

### ISSUE THREAD

**Issue by jackson247** [xibosignage/xibo#82]
Updated to 1.7.0alpha and I can't get past the upgrade prompt. Once I run the checks and agree to the database backup, it shows the following error:

Unable to delete install.php. Please ensure the webserver has permission to unlink this file and retry

The upgrade was a success!

I didn't have an issue with permissions in the past so I don't know why it would be acting up now. When I searched for install.php in my xibo folder it came up with two files. I tried deleting both of them manually and it makes no difference. Am I not deleting the right one(s)? 

When I try to click on anything on the sidebar it just takes me back to the upgrade page. I go through the upgrade again and it shows: "Upgrading from database version 84 to 84" which I'm assuming that I already have this version but it still won't let me delete the install.php

**Comment by alexharrington** [comment]
Perhaps you're seeing cache. Ensure install.php has actually been deleted from your webserver, clear your browser cache and then log back in.

**Comment by shadychris** [comment]
Exactly the same problem here..... Followed the upgrade instructions to the letter. I tweaked my php.ini this time to allow large file uploads but after clicking next to go to the second stage I get the unable to delete install.php but upgrade successful message. The weird thing is that the install.php file that got unzipped into my fresh Xibo directory has been deleted!! 
This is not a browser cache issue - tired clearing the cache several times, even tried in a different browser but still the same just get taken back to the Welcome to the Xibo Upgrade  page and nothing in the left sidebar does anything...
Tried restarting Apache - no difference.
Something somewhere didn't recognise that the installer.php file was deleted and it's now permanently stuck in Welcome to Upgrade mode....
